### PR TITLE
Reg 39 add redux store to application

### PR DESCRIPTION
**Pull request by sturmf**
First attempt of review of redux store. Will need more cleanup.
And due to an unknown reason the e2e tests broke again on travis :-(

### CODE REVIEW

**Comment by sturmf** [comment]
Everything is green again so we are finally getting closer to a mergeable version.

File: frontend/lib/store/RegattaStore.dart [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import 'dart:async';
+import 'package:angular2/core.dart';
+import 'package:greencat/greencat.dart';
+import 'package:frontend/models/event.dart';
+import 'package:frontend/services/event_service.dart';
+import 'package:frontend/app_component.dart';
+
+enum ActionType { addEvent, addNewEvent }
+
+/// Actions to be triggered to the app store.
```
Comment (gesellc) [inline frontend/lib/store/RegattaStore.dart]: Do comments with three slashes have any special usage?

**Comment by sturmf** [comment]
I think it starts to look really good. review can start!

File: frontend/lib/store/RegattaStore.dart (hunk 1 above)
Comment (sturmf) [inline frontend/lib/store/RegattaStore.dart]: Yes they are doc comments, see: https://www.dartlang.org/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types

**Comment by gesellc** [comment]
Maybe this would be a good time to draw something like a design sketch in confluence?

**Pull request merged**